util/topgen/strong_random.py

```diff
@@ -171,8 +170,10 @@ def test_long_run(self):
                 if max_run_len < current_run_len:
                     max_run_len = current_run_len
 
-            results.append(max_run_len < self.LONG_RUN_THRESHOLD)
-        return all(results)
+            if max_run_len >= self.LONG_RUN_THRESHOLD:
```

Comment:
It looks like we can replace this with `if current_run_len >= self.LONG_RUN_THRESHOLD:` and get rid of `max_run_len` altogether.
This was probably my fault. Would it make sense to fix it here?

Reply:
Oh, good point. I've just pushed up a version with that change.